app/models/billing-charge-category.model.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+/**
+ * Model for water.billing_charge_categories
+ * @module BillingChargeCategoryModel
+ */
+
+const { Model } = require('objection')
+
+const BaseModel = require('./base.model.js')
+
+class BillingChargeCategoryModel extends BaseModel {
+  static get tableName () {
+    return 'water.billing_charge_categories'
+  }
+
+  static get relationMappings () {
+    return {
+      chargeElement: {
+        relation: Model.BelongsToOneRelation,
```

Comment:
Think this relationship might need tweaking

```suggestion
        relation: Model.HasManyRelation,
```

Reply:
Yeah just checked, you are right. I'll commit your change.